code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -244,7 +251,51 @@
 				P.desc = "A small condiment pack. The label says it contains [name]."
 				reagents.trans_to(P,10, transfered_by = usr)
 			. = TRUE
-
+		if("pillStyle")
+			var/pill_name = input(usr, "Choose your desired pill style:", "Pill Style")  as null|anything in list("capsule 1", "capsule 2", "capsule 3", "capsule 4", "capsule 5", "capsule 6", "capsule 7", "capsule 8", "capsule 9", "capsule 10", "tablet 1", "tablet 2", "tablet 3", "tablet 4", "tablet 5", "tablet 6", "tablet 7", "tablet 8", "tablet 9", "tablet 10", "tablet 11")
+			switch(pill_name)
+				if("capsule 1")
+					pill_style = 1
+				if("capsule 2")
+					pill_style = 2
+				if("capsule 3")
+					pill_style = 3
+				if("capsule 4")
+					pill_style = 4
+				if("capsule 5")
+					pill_style = 5
+				if("capsule 6")
+					pill_style = 6
+				if("capsule 7")
+					pill_style = 18
+				if("capsule 8")
+					pill_style = 19
+				if("capsule 9")
+					pill_style = 20
+				if("capsule 10")
+					pill_style = 20
```

Comment:
duplicate? also missing styles 21 and 22

Reply:
22 isnt used

Reply:
rather; its not supposed to be